en/release-notes/upgrade-guide-v2.1.md

```diff
@@ -0,0 +1,164 @@
+
+**3D Camera**
+Like `Node`, `Camera` is also divided into two modes. When you add a `Camera` component to a normal 2D node, it becomes a 2D Camera. When you switch `Node` to **3D mode**, `Camera` will also enter **3D mode**, and then you can edit `Camera's` **FOV** and other parameters in **3D mode**.
+
+You can watch switching to **3D Camera** in action: [3D Camera movie](https://www.dropbox.com/s/gncxmsnw8gkomxl/3D%20Camera.mov?dl=0).
+
+**3D model import**
+Currently, __Cocos Creator__ supports importing 3D model formats for a very wide range of **.fbx** files. **.fbx** is the de-facto standard in 3D modeling software. The process of importing is very simple, just drag the **.fbx** model into the **Explorer** and wait a few moments for the import to complete. After the import is completed, you can see that the imported model is a folder that can be expanded in **Explorer**. When importing the model, the editor will automatically parse the content of the model and generate a **Prefab**, a **Grid**, a **Bone Animation** and other resources. See [Import Models](https://docs.cocos2d-x.org/creator/2.1/manual/en/3d/import-model.html)
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_D5EB83DE4EB14ED7F4E0412694594BEA563C6F613AB6CAC04191666891BDD83C_1543326015234_image.png)
+
+
+**Mesh Renderer Component**
+Mesh Renderer is used to draw grid resources. See [Mesh Renderer](https://docs.cocos2d-x.org/creator/2.1/manual/en/3d/mesh-renderer.html).
+
+![](https://github.com/cocos-creator/creator-docs/raw/v2.1/zh/3d/img/mesh_renderer.png)
+
+**Bone Animation Component**
+The **Skeletal Animation** component inherits from the **Animation** component. The **Skeletal Animation** component is not much different from the **Animation** component. The exception is the **Skeletal Animation** component uses a clip that can only be a **Skeletal Animation** clip. The **Skeletal Animation** component is automatically added to the model's Prefab when it is imported. See [Bone Animation Component](https://docs.cocos2d-x.org/creator/2.1/manual/en/3d/skeleton-animation.html) for details.
+
+![](https://github.com/cocos-creator/creator-docs/raw/v2.1/en/3d/img/search-skeleton-animation.png)
+
+For more 3D instructions, please consult the documentation [3D System](https://docs.cocos2d-x.org/creator/2.1/manual/en/3d/index.html).
+
+**Optimize automatic gallery build speed**
+__v2.1__ greatly optimizes the speed of auto-atlas construction. If the atlas has not changed, the results of the last build are used.
+
+**Further improve rendering performance**
+Between __v2.0__ compared to __v1.x__, there are some cases where the engine does not render batches, resulting in an increase in draw calls. __v2.1__ re-optimized these conditions. If the previous **color** or **opacity** cannot be batch processed, **sliced** and **non-sliced** share the same texture and cannot be batched. The problem that the **opacity** 0 node still occupies the draw call is solved.
```

Comment:
1. It is best to have a blank line between the title and the body
2. draw calls -> Draw Call
3. If -> For example

Reply:
2. ok

3. ok